src/main/resources/META-INF/plugin.xml

```diff
@@ -139,22 +144,37 @@
                          implementationClass="org.elm.ide.inspections.ElmIncompletePatternInspection"/>
 
 
+        <!-- Intentions -->
+
+        <intentionAction>
```

Comment:
These are the real changes to the `plugin.xml`. When you register an intention action like this, you also need to provide documentation HTML and before/after examples. Failure to do so will result in IntelliJ disabling your plugin at launch time. So I added a new test `ElmIntentionDocsTest` (based on intellij-rust), which verifies that the documentation exists.